client/src/components/Chat/ChatHeader.js

```diff
@@ -112,7 +113,7 @@ const ChatHeader = props => {
       <div className="chatHeaderRight">
         <Typography component="div">
           <Grid component="label" container alignItems="center" spacing={1}>
-            <Grid item>Off</Grid>
+            <Grid item>Original Language</Grid>
             <Grid item>
               <AntSwitch checked={state.checkedC} onChange={handleChange} name="checkedC" />
```

Comment:
right now it seems like you're storing the control checked state value in two places, in the ChatHeader state and also above in the Chat component. it could be refactored to just store the checked state in Chat and pass that down to here. this is residual code from where I copied the material ui sample but it should be cleaned up.

Reply:
I tried it but Im unable to flip the Switch if I try to store checked state in the parent and pass it down as props here. Pretty much all material UI samples are using states to toggle the switch. Im not sure how I could control it from above Chat component. Let me know if you have suggestions on this.